src/site/content/en/blog/websocketstream/index.md

```diff
@@ -0,0 +1,332 @@
+---
+title: WebSocketStream—Integrating Streams with the WebSocket API
```

Comment:
```suggestion
title: WebSocketStream: integrating streams with the WebSocket API
```

Reply:
It needs to be on a new line due to the colon, but yes.